ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/Genesis/Tests/LongRangeAttack.hs

```diff
@@ -79,4 +85,13 @@ prop_longRangeAttack honestFreq advFreqs = do
     isHonestTestHeaderHash :: HeaderHash TestBlock -> Bool
     isHonestTestHeaderHash = all (0 ==) . unTestHash
 
-    scheduleConfig = defaultPointScheduleConfig
+    scheduleConfig = PointScheduleConfig {pscTickDuration}
+
+    pscTickDuration | useIOSim = 0.1
+                    | otherwise = 0.001
```

Comment:
```suggestion
    scheduleConfig | useIOSim = defaultPointScheduleConfig
                   | otherwise = defaultPointScheduleConfig {pscTickDuration=0.001}
```